### PR TITLE
Remove insecure rng

### LINKED ISSUE #173: Insecure random values should be opt-in
Right now, this module will automatically fall back to `Math.random()` in browser environments that don't support CSPRNG use, essentially failing open.

To operate securely, it should instead fail hard with an error, and only allow falling back to an insecure random source if the developer explicitly indicates that this is acceptable.

### PR DESCRIPTION
**Pull request by ctavan**
Closes #294 and fixes #173

### CODE REVIEW

**Comment by ctavan** [comment]
BTW as expected this now fails in IE 10 where no crypto.getRandomValues() is available:

Screenshot 2020-01-20 23 35 52

**Comment by ctavan** [comment]
@broofa do you think that, given the now-wide-spread support for CSRNGs in all platforms, we still need the wiki page over at https://github.com/uuidjs/uuid/wiki#no-secure-rng-error ?

**Comment by broofa** [comment]
Go ahead and kill that wiki page.

**Comment by ctavan** [comment]
@broofa did you close this PR on purpose?

**Comment by broofa** [comment]
I did not.  Sorry about that!

File: README_js.md [hunk 1]
```diff
@@ -123,8 +123,66 @@ npm run build
...
+
+```javascript --run v4
+const buffer = new Array();
+uuid.v4(null, buffer, 0); // RESULT
+uuid.v4(null, buffer, 16); // RESULT
+```
+
 ### Version 1 (Timestamp + Node)
 
+⚠️⚠️⚠️ **Please make sure to check whether you really need the timestamp properties of Version 1
```
Comment (broofa) [inline README_js.md]: Would be good to have a link of some sort that provides information to help with readers' v1 vs v4 decision making.

Comment (ctavan) [reply README_js.md]: I've added a link to the standard lib proposal. I think that covers the reasoning.

**Pull request merged**